languageFactories/wdl-draft2/src/main/scala/languages/wdl/draft2/WdlDraft2LanguageFactory.scala

```diff
@@ -122,7 +125,7 @@ class WdlDraft2LanguageFactory(override val config: Config) extends LanguageFact
       _ <- enabledCheck
       namespace <- WdlNamespace.loadUsingSource(workflowSource, None, Some(importResolvers map resolverConverter)).toChecked
       womBundle <- namespace.toWomBundle
-    } yield womBundle
+    } yield womBundle.copy(resolvedImportRecords = womBundle.resolvedImportRecords ++ workflowSourceOrigin)
```

Comment:
Cool!

Reply:
Good point. I added a similar method in `WomBundle`